lib/improver/wind_downscaling.py

```diff
@@ -89,33 +86,39 @@ def __init__(self, u_href, h_ref, z_0, mask):
         self.z_0 = z_0
         self.mask = mask
 
-    def calc_ustar(self):
+        array_sizes = [np.size(u_href), np.size(h_ref), np.size(z_0),
+                       np.size(mask)]
+
+        if not all(x == array_sizes[0] for x in array_sizes):
+            raise ValueError('Different size input arrays u_href, h_ref, z_0, '
+                             'mask')
+
+    def process(self):
         """Function to calculate the friction velocity.
 
-        ustar = K * u_href / ln(h_ref / z_0)
+        ustar = K * (u_href / ln(h_ref / z_0))
 
         where ustar is the friction velocity, K is Von Karman's
         constant, u_ref is the wind speed at the reference height,
         h_ref is the reference height and z_0 is the vegetative
         roughness length.
 
         Returns:
-            ustar (2D array float):
-                friction velocity
+            ustar (ndarray):
+                A 2D array of friction velocities
 
         """
         ustar = np.full(self.u_href.shape, RMDI)
         ustar[self.mask] = (
-            VONKARMAN * (
-                self.u_href[self.mask] /
-                np.log(self.h_ref[self.mask] / self.z_0[self.mask])
-            )
+                VONKARMAN * (
+                    self.u_href[self.mask] /
+                    np.log(self.h_ref[self.mask] / self.z_0[self.mask])
+                )
         )
         return ustar
 
 
 class RoughnessCorrectionUtilities(object):
```

Comment:
Sorry about all the minor reformatting in the class RoughnessCorrectionUtilities - I'm not quite sure how I managed to do this but it's just mainly putting in a space either side of operators like  /  *  + (which does look better). No code has actually changed.